src/js/view/controls/components/menu/settings-menu.js

```diff
@@ -188,9 +188,15 @@ class SettingsMenu extends Menu {
             });
             captionsSettingsItems.push(resetItem);
             captionsSettingsMenu.setMenuItems(captionsSettingsItems);
+            if (isReset) {
+                resetItem.statusEl.innerHTML = 'Reset Captions Successful';
```

Comment:
Might be better to use [`textContent`](https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent#differences_from_innertext) here and below rather than `innerHTML`